src/EnergyPlus/ScheduleManager.hh

```diff
@@ -121,6 +121,14 @@ namespace ScheduleManager {
         Num
     };
 
+    enum class Clusive
```

Comment:
I didn't know what else to call this `enum`. Oh, here's a fun fact! Why am I using an `enum` and not a `bool`? Because apparently `string` and `string_view` can both upcast to `bool`! Empty string means `false`, everything else is `true`. So if you have an overloaded function with an argument that can be either a `string` or a `bool`, only the `bool` will get called! At least I think that this is what is happening. In any case, `string` does not upcase to `enum` so that sidesteps that.